source/short.rst

```diff
@@ -103,8 +103,12 @@ Mouse in "Content" Tab
 |                    +---------------------+----------------------------------------------+
 |                    |   "delete"          | - delete selected file                       |
 |                    +---------------------+----------------------------------------------+
-|                    |   "delete group     | - delete selected file and its relatives     |
+|                    |   "delete group"    | - delete selected file and its relatives     |
```

Comment:
by the way, isn't *delete group* gone since the "`projectView`" refactory of some time ago??